content/concept-amendments.md

```diff
@@ -209,6 +225,37 @@ Fixes a minor bug in transaction processing that causes failures when trying to
 
 The fix1373 amendment corrects the issue so that the paths are properly prepared and payments can use them. It also disables some inappropriate paths that are currently allowed, including paths whose [steps](concept-paths.html#path-specifications) include conflicting fields and paths that loop through the same object more than once.
 
+## fix1512
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 6C92211186613F9647A89DFFBAB8F94C99D4C7E956D495270789128569177DA1 | Enabled   |
+
+Fixes a bug in transaction processing that causes some invalid [PaymentChannelClaim][] transactions to fail with the wrong error code. Without this amendment, the transactions have a `tec`-class result code despite not being included in a ledger and therefore not paying the [transaction cost](concept-transaction-cost.html).
+
+With this amendment, the transactions fail with a more appropriate result code, `temBAD_AMOUNT`, instead.
+
+## fix1523
+
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| B9E739B8296B4A1BB29BE990B17D66E21B62A300A909F25AC55C22D6C72E1F9D | Enabled   |
+
+Adds tracking by destination account to [escrows](concept-escrow.html). Without this amendment, pending escrows are only tracked by sender. This amendment makes it possible to look up pending escrows by the destination address using the [`account_objects` command](reference-rippled.html#account-objects), excluding any pending escrows that were created before this amendment became enabled. This amendment also makes [EscrowCreate transactions][] appear in the destination's transaction history, as viewed with the [`account_tx` command](reference-rippled.html#account-tx).
```

Comment:
Sounds like this means we should add a "Look up escrows by receiver" section to the Escrow tutorial? (Not for this PR, though.)

Reply:
Yep, the fact that we couldn't write that section originally is what inspired this amendment.

But that's a good reminder because I don't think we have a JIRA task on the backlog for said update.

Reply:
Created [DOC-1165](https://ripplelabs.atlassian.net/browse/DOC-1165) to track it.